src/repr/src/scalar.rs

```diff
@@ -1411,6 +1411,24 @@ impl<'a> ScalarType {
     }
 
     /// Returns number of layers (akin to array dimensions) on a
```

Comment:
Looks like this line got frankenstein'ed onto this code.

Reply:
whoops, merge conflict resolution. good catch 🕵️‍♂️